### DISCUSSION TITLE
I want to move a specific location to the center on the map screen.

### DISCUSSION THREAD

**Discussion by SeanSoonilLim** [electricitymaps/electricitymaps-contrib#4959] [Ideas] [answered]
How's everybody doing?

I'm reviewing the service using `electricitymaps`.

When I access `https://app.electricitymaps.com/`, I can see where I am currently.

Is there a way to move the map's focus by entering a specific zone, latitude, and longitude as query parameters on the URL?

Thank you.

**Comment by VIKTORVAV99** [comment]
I don't think there is right now but it's a good suggestion and we will keep it in mind when we start looking into new features to add.

**Comment by VIKTORVAV99** [comment, marked as answer]
We just merged https://github.com/electricitymaps/electricitymaps-contrib/pull/5046 which should enable zooming or traveling to a specific area on the map based on the zone id. 

It just need to be deployed to production before it is generally available.
